source/server/infrastructure/router.coffee

```diff
@@ -54,13 +55,23 @@ class Space.eventSourcing.Router extends Space.messaging.Controller
       @eventBus.subscribeTo @initializingMessage, (event) =>
         @log.info("#{this}: Creating new #{@eventSourceable} with event
                   #{event.typeName()}\n", event)
-        eventSourceable = @_handleDomainErrors(-> new @eventSourceable event)
+        eventSourceable = @_handleDomainErrors(->
+          instance = new @eventSourceable(event.sourceId)
+          @_injectDependencies(instance)
```

Comment:
i don't think the wrapper function is necessary. `this.injector.injectInto(instance)` has same length and is even more readable :wink: